apps/patient/src/components/PickupOptions.tsx

```diff
@@ -187,11 +187,6 @@ export const PickupOptions = ({
           {t.showMore}
         </Button>
       ) : null}
-      {!loadingMore && showingAllPharmacies ? (
-        <Text color="gray.500" textAlign="center">
-          {t.showingAll}
```

Comment:
patients are clicking this non-clickable text to get more pharmacies, jomi suggested removing